src/components/Person/Person.jsx

```diff
@@ -1 +1,30 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+  const partner = sex === 'f'
+    ? 'husband'
+    : 'wife';
+
+  const married = isMarried
+    ? `${partnerName} is my ${partner}`
+    : 'I am not married';
+
+  const personAge = !age
+    ? null
+    : <p className="Person__age">{`I am ${age}`}</p>;
```

Comment:
IMO, it's better to do all conditional rendering inside JSX, because now I need to jump between variables and remember what will be rendered and where